redfish-core/lib/message_registries.hpp

```diff
@@ -56,16 +57,7 @@ inline void handleMessageRegistryFileCollectionGet(
 
     for (const char* memberName :
          std::to_array({"Base", "TaskEvent", "ResourceEvent", "OpenBMC",
-                        "BiosAttributeRegistry"}))
-    {
-        nlohmann::json::object_t member;
```

Comment:
this was a dub

Reply:
Yes, this PR also removes the duplicated output entries.
Prior to the fix, we got:
```
curl -k -X GET https://${bmc}/redfish/v1/Registries/
{
...
  "Members": [
    {
      "@odata.id": "/redfish/v1/Registries/Base"
    },
....
    {
      "@odata.id": "/redfish/v1/Registries/Base"
    },
```